src/include/homestore/chunk_selector.h

```diff
@@ -0,0 +1,30 @@
+/*********************************************************************************
+ * Modifications Copyright 2017-2019 eBay Inc.
```

Comment:
New files can have a new copyright date (2023)

Reply:
got it!